BotSetup.py

```diff
@@ -1,30 +1,115 @@
 import os
-import sqlite3
+import time
 from Config import Config
 from Logger import LogLevel, Logger
+from sqlalchemy import create_engine, select, Table, Column, Integer, text, DateTime, String, URL, asc, desc
+from sqlalchemy.sql import func
+from sqlalchemy.inspection import inspect
+from sqlalchemy.orm import declarative_base, Session
+from sqlalchemy_easy_softdelete.mixin import generate_soft_delete_mixin_class
+from sqlalchemy_easy_softdelete.hook import IgnoredTable
+from datetime import datetime
+from BotDatabaseSchema import *
 
-DATABASE_VERSION=1
 
 def SetupDatabases():
-    DatabaseExists:bool = os.path.exists(Config.GetDBFile())
-    Logger.Log(LogLevel.Notice, "Loading database for scam bot setup")
-    con = sqlite3.connect(Config.GetDBFile())
-    cursor = con.cursor()
-    CurrentVersion:int = cursor.execute("PRAGMA user_version").fetchone()[0]
-    if (CurrentVersion != 0):
-        # Version updating for the database
-        if (CurrentVersion != DATABASE_VERSION):
-            Logger.Log(LogLevel.Error, f"Please run the database update script from version {CurrentVersion} to {DATABASE_VERSION}")
-        else:
-            Logger.Log(LogLevel.Debug, f"Database version is currently {CurrentVersion}")
-            con.close()
-            return
-
-    cursor.execute(f"PRAGMA user_version = {DATABASE_VERSION}")
-    cursor.execute("CREATE TABLE if not EXISTS banslist(Id, BannerName, BannerId, Date)")
-    cursor.execute("CREATE TABLE if not EXISTS servers(Id, OwnerId, Activated)")
-    con.commit()
-    con.close()
-    Logger.Log(LogLevel.Notice, "Created the bot databases!")
-
-SetupDatabases()
\ No newline at end of file
+
+    ConfigData=Config()
```

Comment:
This doesn't need to be initialized here, the functions being called are static methods.

Reply:
Will remove